python/desc/twinkles/io.py

```diff
@@ -0,0 +1,25 @@
+from __future__ import absolute_import, print_function, division
```

Comment:
Although this code is imported into the `desc.twinkles` namespace, it might be a good idea to rename the file to something less generic.  There already is an `io.py` module that's part of the standard python2 distribution.